src/intro.md

```diff
@@ -6,6 +6,6 @@ updated by many contributors over the Internet.
 
 Check the Github site to add new contents or fix typos:
 
-* Github: https://github.com/radareorg/radare2book
-* Online: https://radareorg.github.io/radare2book/index.html
-
+<!-- Fix links from not being links when rendered by mdbook -->
```

Comment:
Can you remove those comments?

Reply:
No problem. I was just afraid it would be tempting for someone later to 'simplify' those links back to plain text.